roles/sap_netweaver_preconfigure/defaults/main.yml

```diff
@@ -5,7 +5,7 @@
 # Perform an assertion run:
 sap_netweaver_preconfigure_assert: false
 
-# In case of an assertion run, if set to "yes", the role will abort for any assertion error:
+# In case of an assertion run, if set to "true", the role will abort for any assertion error:
```

Comment:
Let's also change the double quotes for single ones:
`# In case of an assertion run, if set to 'true', the role will abort for any assertion error:`